stable_baselines3/common/vec_env/dummy_vec_env.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, env_fns: List[Callable[[], gym.Env]]):
         self.buf_rews = np.zeros((self.num_envs,), dtype=np.float32)
         self.buf_infos: List[Dict[str, Any]] = [{} for _ in range(self.num_envs)]
         self.metadata = env.metadata
+        
+        self.seed = [None for _ in range(len(self.envs))]  # seed to be used in the next env.reset()
```

Comment:
You should rename it to `_seeds` otherwise it will conflict with the `seed()` method.